README.md

```diff
@@ -531,96 +567,108 @@ public class Rectangle implements Shape {
 
 When we talk about an interface and define capabilities that we promise to provide, we are talking about establishing a contract about what the object can do.
 
-For example, the following can all be animals: cat, dog, hippo, meerkat. However, only a cat or dog would lick people. In this case we could make Animal an abstract class where the animals can inherit certain characteristics, and LicksPeople as an interface, which the cat and the dog would implement.
+For example, the following can all be animals: cat, dog, hippo, meerkat. However, only a cat or dog would lick people. In this case we could make `Animal` an abstract class where the animals can inherit certain characteristics, and `PeopleLicker` as an interface, which the cat and the dog would implement.
 
 # Git Bash
 
-Git is a command line application adjacent to code. It excels as a Version Control System. There are some other systems you can use, but Git is the best. There are many Graphical User Interfaces (GUIs) for Git e.g. IntelliJ. Bash (Bourne Again SHell) is a program that facilitates the running of applications.
-
-Git has directories and repositories. The directories are the structure of files and folders and the repositories track changes and form the basis of the version control system.
-
-## Useful Git Commands
-
-Some Git commands need a 'Git' in front of them as they are Git specific.
-
-history  (shows the command line history)\
-cd  (change directory)\
-ls  (list contents of current directory)\
-git status  (checking the state of a repository)\
-git init  (creates an empty repository within the current directory)\
-git add  (adds files where the names succeed the key word)\
-git commit  (commits the files to the repository)\
-\-m  (allows you to add a comment within "" to the commit)\
-nano  (text editor)\
-cat  (concatenates/combines - shows you the file)\
-cp  (copy file)\
-mv  (move and rename file or directory)\
-/  (the highest directory you can navigate to)\
-~  (the home directory)\
-.  (the current directory)\
-..  (the directory above the current one)\
-\-  (the previous directory you navigated to)\
-rm  (remove file - be VERY careful with this one!)
+Git is an application on your computer. It does not come with a pretty interface by default, so you will have to use the command line to use it. It is an application adjacent to code - I.e. it is not directly related to programming, but there are very few professional programrs who don't use it. It excels as a 'Version Control System'. There are some other systems you can use, but Git is the best and is dominating in terms of popularity. There are many Graphical User Interfaces (GUIs) for Git e.g. IntelliJ. When you install Git on a Windows computer you generally also get a program called 'Git Bash'. Bash (Bourne Again SHell) is a program that facilitates the running of command line applications.
```

Comment:
Is "programrs" a standard abbreviation? (I also typed this into Google, but I'm none the wiser!).

Reply:
Nope! It's just be being silly.

I did a global search-and-replace of "programme" with "program", which turned "programmer" into "programr".